Tests/Shared/Stub/HttpRequestRunnerStub.swift

```diff
@@ -5,9 +5,12 @@ import FoundationNetworking
 #endif
 
 public class HttpRequestRunnerStub {
+    // Runner either uses queue, or always returns the same response.
     private var responseQueue: [HttpResponse] = []
+    private var responseToAlwaysReturn: HttpResponse?
```

Comment:
How about renaming this variable `responseToAlwaysReturn`, always return (what) ?